front/src/views/recovery-password/recovery-password.view.html

```diff
@@ -31,130 +31,13 @@ <h4>{{$t('recoveryPassword.title')}}</h4>
-                  </v-btn>
-                </v-col>
-              </v-row>
-            </v-form>
-            <div class="text-center text-normal">
-              {{$t('recoveryPassword.note')}}
-            </div>
-          </v-card>
+        <mmt-recovery-step-1/>
         </v-stepper-content>
         <v-stepper-content step="2">
-          <v-card class="recovery-card">
-          <div class="text-center text-normal">
-            {{$t('recoveryPassword.successfulСonfirmation')}}
-          </div>
-          <v-form ref="form" v-model="valid" >
-            <div class="d-flex justify-center">
-              <mmt-text-input
-                :placeholder="$t('recoveryPassword.createPassword')"
-                :mmt-label="$t('recoveryPassword.password')"
-                class="input-field"
-                rounded
-                filled
-                dense
-                :type="showPassword  ? 'text' : 'password'"
-                :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
-                @click:append="showPassword = !showPassword"
-                :rules="[rules.isEmpty, rules.minPassLen]"
-                v-model="password"
-                required
-              />
-            </div>
-            <div class="d-flex justify-center">
-              <mmt-text-input
-                :placeholder="$t('recoveryPassword.repeatPassword')"
-                :mmt-label="$t('recoveryPassword.confrimPassword')"
-                class="input-field"
-                rounded
-                filled
-                dense
-                :type="showPasswordConfirmation ? 'text' : 'password'"
-                :append-icon="showPasswordConfirmation ? 'mdi-eye' : 'mdi-eye-off'"
-                @click:append="showPasswordConfirmation = !showPasswordConfirmation"
-                :rules="[rules.isEmpty, rules.minPassLen, rules.samePass]"
-                v-model="passwordConfirmation"
-                @input="onInput"
-                required
-              />
-            </div>
-            <v-row cols="2" justify="center">
-              <v-col justify="center">
-                <v-btn
-                  class="mmt-btn btn-step-2"
-                  rounded
-                  color="primary"
-                  :disabled="!valid"
-                  @click="sendPassword"
-                  >{{$t('recoveryPassword.save')}}
-                </v-btn>
-              </v-col>
-              <v-col justify="center">
-                <v-btn
-                  rounded
-                  outlined
-                  color="accent lighten-1"
-                  class="mmt-btn btn-step-2"
-                  dense
-                  @click="$router.push('/');"
-                  >{{$t('recoveryPassword.back')}}
-                </v-btn>
-              </v-col>
-            </v-row>
-          </v-form>
-        </v-card>
+          <mmt-recovery-step-2 @stepDone="step = 3" @stepNotDone="step = 2"/>
```

Comment:
- currentStep
- js handle function could be better than template code
- stepDone and stepNotDone confuses me. it is more appreciate  to use just stepDone for each step component
- might, it is time to understand presentation and container components. For example, your step components are only presentation, and your recovery-password is container. it is just thought. read article and decide. 

https://medium.com/@kanby/%D0%BF%D1%80%D0%B5%D0%B7%D0%B5%D0%BD%D1%82%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D0%B9-%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82-%D0%B8-%D0%BA%D0%BE%D0%BD%D1%82%D0%B5%D0%B9%D0%BD%D0%B5%D1%80-%D0%B2-react-f0e118480809